.github/workflows/publish-sdk.yaml

```diff
@@ -8,10 +8,10 @@ env:
 
 jobs:
   lint-test-sdk:
-    runs-on: macos-latest
+    runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
```

Comment:
Was hitting weird git issues that updating this resolved 🤷‍♂️

Reply:
Yeah I've randomly had to bump this before too. No idea 🤷‍♂️